### DISCUSSION TITLE
Is there a documentation?

### DISCUSSION THREAD

**Discussion by Laserjones** [khoih-prog/LittleFS_Mbed_RP2040#3] [Q&A] [answered]
Hello Khoih,
is there some kind of documentation or reference for this library regarding usage, functions and parameters? So far I only found your two examples, which help, but don't explain all the details.
Best regards,
Joerg

**Comment by khoih-prog** [comment, marked as answer]
Check the [**Features**](https://github.com/khoih-prog/LittleFS_Mbed_RP2040#features) and see there is nothing special and it's very easy to use the library.

> The filesystem access uses normal [POSIX APIs](https://www.tutorialspoint.com/c_standard_library/stdio_h.htm) or [mbed FileSystem APIs](https://os.mbed.com/docs/mbed-os/v6.10/apis/file-system-apis.html)

But it's always better to `learn-by-example`

But we'd all appreciate that you'd write and add the documentation as you'd think necessary here.
